2024-04-11 14:28:28,478 INFO    StreamThr :30120 [internal.py:wandb_internal():86] W&B internal server running at pid: 30120, started at: 2024-04-11 14:28:28.476887
2024-04-11 14:28:28,479 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: status
2024-04-11 14:28:28,480 INFO    WriterThread:30120 [datastore.py:open_for_write():87] open: c:\AIO24\Daily_Practice\Day38\wandb\run-20240411_142822-kkgmiuj4\run-kkgmiuj4.wandb
2024-04-11 14:28:28,481 DEBUG   SenderThread:30120 [sender.py:send():379] send: header
2024-04-11 14:28:28,481 DEBUG   SenderThread:30120 [sender.py:send():379] send: run
2024-04-11 14:28:29,526 INFO    SenderThread:30120 [dir_watcher.py:__init__():211] watching files in: c:\AIO24\Daily_Practice\Day38\wandb\run-20240411_142822-kkgmiuj4\files
2024-04-11 14:28:29,526 INFO    SenderThread:30120 [sender.py:_start_run_threads():1124] run started: kkgmiuj4 with start time 1712820507.580726
2024-04-11 14:28:29,537 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: check_version
2024-04-11 14:28:29,538 DEBUG   SenderThread:30120 [sender.py:send_request():406] send_request: check_version
2024-04-11 14:28:34,543 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: status_report
2024-04-11 14:28:34,548 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: run_start
2024-04-11 14:28:34,558 DEBUG   HandlerThread:30120 [system_info.py:__init__():26] System info init
2024-04-11 14:28:34,558 DEBUG   HandlerThread:30120 [system_info.py:__init__():41] System info init done
2024-04-11 14:28:34,558 INFO    HandlerThread:30120 [system_monitor.py:start():194] Starting system monitor
2024-04-11 14:28:34,559 INFO    SystemMonitor:30120 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-04-11 14:28:34,559 INFO    HandlerThread:30120 [system_monitor.py:probe():214] Collecting system info
2024-04-11 14:28:34,567 INFO    SystemMonitor:30120 [interfaces.py:start():190] Started cpu monitoring
2024-04-11 14:28:34,568 INFO    SystemMonitor:30120 [interfaces.py:start():190] Started disk monitoring
2024-04-11 14:28:34,568 DEBUG   HandlerThread:30120 [system_info.py:probe():150] Probing system
2024-04-11 14:28:34,569 INFO    SystemMonitor:30120 [interfaces.py:start():190] Started gpu monitoring
2024-04-11 14:28:34,569 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:28:34,570 INFO    SystemMonitor:30120 [interfaces.py:start():190] Started memory monitoring
2024-04-11 14:28:34,570 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:28:34,580 INFO    SystemMonitor:30120 [interfaces.py:start():190] Started network monitoring
2024-04-11 14:28:34,581 DEBUG   HandlerThread:30120 [gitlib.py:_init_repo():56] git repository is invalid
2024-04-11 14:28:34,582 DEBUG   HandlerThread:30120 [system_info.py:probe():198] Probing system done
2024-04-11 14:28:34,582 DEBUG   HandlerThread:30120 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.22621-SP0', 'python': '3.11.7', 'heartbeatAt': '2024-04-11T07:28:34.569574', 'startedAt': '2024-04-11T07:28:22.598412', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '<python with no main file>', 'codePathLocal': None, 'host': 'Keno', 'username': 'namti', 'executable': 'c:\\Users\\namti\\anaconda3\\envs\\aio24_env\\python.exe', 'cpu_count': 8, 'cpu_count_logical': 16, 'cpu_freq': {'current': 3000.0, 'min': 0.0, 'max': 3000.0}, 'cpu_freq_per_core': [{'current': 3000.0, 'min': 0.0, 'max': 3000.0}], 'disk': {'/': {'total': 953.0654792785645, 'used': 772.5243644714355}}, 'gpu': 'NVIDIA GeForce RTX 2060 with Max-Q Design', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 2060 with Max-Q Design', 'memory_total': 6442450944}], 'memory': {'total': 15.41949462890625}}
2024-04-11 14:28:34,582 INFO    HandlerThread:30120 [system_monitor.py:probe():224] Finished collecting system info
2024-04-11 14:28:34,582 INFO    HandlerThread:30120 [system_monitor.py:probe():227] Publishing system info
2024-04-11 14:28:34,582 DEBUG   HandlerThread:30120 [system_info.py:_save_conda():207] Saving list of conda packages installed into the current environment
2024-04-11 14:28:35,587 INFO    Thread-43 :30120 [dir_watcher.py:_on_file_created():271] file/dir created: c:\AIO24\Daily_Practice\Day38\wandb\run-20240411_142822-kkgmiuj4\files\conda-environment.yaml
2024-04-11 14:28:39,560 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:28:39,560 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:28:42,301 DEBUG   HandlerThread:30120 [system_info.py:_save_conda():222] Saving conda packages done
2024-04-11 14:28:42,302 INFO    HandlerThread:30120 [system_monitor.py:probe():229] Finished publishing system info
2024-04-11 14:28:42,302 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: status_report
2024-04-11 14:28:42,302 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: keepalive
2024-04-11 14:28:42,303 DEBUG   SenderThread:30120 [sender.py:send():379] send: files
2024-04-11 14:28:42,303 INFO    SenderThread:30120 [sender.py:_save_file():1390] saving file wandb-metadata.json with policy now
2024-04-11 14:28:42,642 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: python_packages
2024-04-11 14:28:42,643 DEBUG   SenderThread:30120 [sender.py:send_request():406] send_request: python_packages
2024-04-11 14:28:42,643 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: stop_status
2024-04-11 14:28:42,644 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: internal_messages
2024-04-11 14:28:42,644 DEBUG   SenderThread:30120 [sender.py:send_request():406] send_request: stop_status
2024-04-11 14:28:42,677 INFO    Thread-43 :30120 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\AIO24\Daily_Practice\Day38\wandb\run-20240411_142822-kkgmiuj4\files\conda-environment.yaml
2024-04-11 14:28:42,677 INFO    Thread-43 :30120 [dir_watcher.py:_on_file_created():271] file/dir created: c:\AIO24\Daily_Practice\Day38\wandb\run-20240411_142822-kkgmiuj4\files\requirements.txt
2024-04-11 14:28:42,677 INFO    Thread-43 :30120 [dir_watcher.py:_on_file_created():271] file/dir created: c:\AIO24\Daily_Practice\Day38\wandb\run-20240411_142822-kkgmiuj4\files\wandb-metadata.json
2024-04-11 14:28:43,386 DEBUG   SenderThread:30120 [sender.py:send():379] send: telemetry
2024-04-11 14:28:43,418 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: log_artifact
2024-04-11 14:28:43,419 DEBUG   SenderThread:30120 [sender.py:send_request():406] send_request: log_artifact
2024-04-11 14:28:43,433 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,434 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,434 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,435 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,436 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,436 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,436 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,436 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,437 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,437 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,437 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,438 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,439 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,439 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,440 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,440 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,440 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,440 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,441 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,441 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,442 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,442 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,442 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,443 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,443 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,443 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,443 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,443 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,444 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,444 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,444 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,445 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,445 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,445 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,445 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,446 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,446 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,446 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,447 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,447 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,447 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: partial_history
2024-04-11 14:28:43,689 INFO    Thread-43 :30120 [dir_watcher.py:_on_file_created():271] file/dir created: c:\AIO24\Daily_Practice\Day38\wandb\run-20240411_142822-kkgmiuj4\files\media\table\Dataset_0_4f0dfea7efcf3ff09626.table.json
2024-04-11 14:28:43,690 INFO    Thread-43 :30120 [dir_watcher.py:_on_file_created():271] file/dir created: c:\AIO24\Daily_Practice\Day38\wandb\run-20240411_142822-kkgmiuj4\files\media\table
2024-04-11 14:28:43,690 INFO    Thread-43 :30120 [dir_watcher.py:_on_file_created():271] file/dir created: c:\AIO24\Daily_Practice\Day38\wandb\run-20240411_142822-kkgmiuj4\files\media
2024-04-11 14:28:43,967 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:28:43,967 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:28:44,450 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: poll_exit
2024-04-11 14:28:48,384 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:28:48,384 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:28:49,506 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: keepalive
2024-04-11 14:28:52,787 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:28:52,787 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:28:54,559 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: keepalive
2024-04-11 14:28:57,203 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:28:57,203 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:28:59,618 DEBUG   HandlerThread:30120 [handler.py:handle_request():146] handle_request: keepalive
2024-04-11 14:29:01,613 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:29:01,613 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:29:06,023 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:29:06,023 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:29:10,506 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:29:10,506 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:29:14,923 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:29:14,925 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:29:19,339 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-04-11 14:29:19,339 ERROR   gpu       :30120 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
